storage_service/administration/views.py

```diff
@@ -25,6 +26,29 @@ def settings_edit(request):
     return render(request, 'administration/settings_form.html', locals())
 
 
+########################## VERSION ########################
+
+def get_ss_version():
+    try:
+        from storage_service import __version__ as ss_version
```

Comment:
If someone deletes `__version__` then that's probably a bug we want to be aware of by letting the exception bubble up.  This import can probably be moved to the top of the file (which may mean this function gets folded into `version_view`).